pkgs/build-support/fetchtorrent/tests.nix

```diff
@@ -1,47 +1,58 @@
-{ testers, fetchtorrent, ... }:
+{
+  lib,
+  testers,
+  fetchtorrent,
+  ...
+}:
 
 let
-  wired-cd.meta.license = [
-    # track 1, 4 and 11
-    {
-      spdxID = "CC NC-SAMPLING+ 1.0 Deed";
-      fullName = "NonCommercial Sampling Plus 1.0 Generic";
-      url = "https://creativecommons.org/licenses/nc-sampling+/1.0/";
-      free = false; # for noncommercial purposes only
-    }
-    # the rest
-    {
-      spdxID = "CC SAMPLING+ 1.0 Deed";
-      fullName = "Sampling Plus 1.0 Generic";
-      url = "https://creativecommons.org/licenses/sampling+/1.0/";
-      free = true; # no use in advertisement
-    }
-  ];
+  sintel.meta = {
```

Comment:
tl;dr if you can think of something smaller, that'd be great, but this is ok.

This will be fetched as many times as the *drv* hash changes due to `invalidateFetcherByDrvHash`.
That's every time *anything* about the implementation is different.
For this reason the size of our choice here is perhaps somewhat significant.
Smaller is better, but it won't be the worst offender, and this copy of Sintel's at 129MB seems acceptable.
Another alleviating factor is that the number of copies that ends up in the cache will be smaller than the number of unique store paths because it gets deduplicated a bit, by *compressed nar* hash, which is an imperfect strategy, but surely reduces the number of copies we'll end up storing.

Reply:
@marcin-serwin yes, I tried a few different IA torrents, but none of them would download for me. I'm guessing the issue is most of them are only seeded by a HTTP source rather than a bunch of other people running Bittorrent clients, and neither transmission nor rqbit support HTTP seeds.